docs/content/release_notes/whats_new.md

```diff
@@ -4,10 +4,33 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
 
+
 ## New support for policy-based replication
 
 This version adds support for policy-based replication that was introduced in IBM Spectrum Virtualize 8.5.2 release. Policy-based replication provides a simplified configuration and management of asynchronous replication between two system. To see if your specific product is supported and for more information, see **What's new** > **Getting started with policy-based replication** within your Spectrum Virtualize product documentation on [IBM Documentation](https://www.ibm.com/docs).
 
+## Enhanced dynamic host definition
+
+The IBM® block storage CSI driver 1.11.0 host definition feature now supports the following:
+
+- **CSI Topology feature**
+
+    For more information, see [Configuring for CSI Topology](../configuration/confiugring_toplogy.md).
+
+- **Dynamically configuring host ports**
+
+     Changes in host port hierarchy are now identified automatically updated. For more information, see [Configuring the host definer](../configuration/configuring_hostdefiner.md).
+
+- **I/O groups**
+
+    By default the host definer now creates all definitions across all possible I/O groups. Additionally, an optional label available in order to specify which I/O group should be used on a specific node. For more information, see [Using dynamic host definition](../configuration/configuring_hostdefiner.md).
```

Comment:
we don't have any information about the I/O group feature in the confiugring_toplogy.md page

Reply:
what do you expect to be mentioned there? is it part of topology? 
we don't currently have any i/o group information as part of topology so there is nothing to just add to.

Reply:
that if we are giving the user a link for more information, it should be something relevant to the subject, maybe point him to an I/O group page that we have in our documentation or even in IBM knowledge center

Reply:
I fixed the link to point to the label section now